05-networks/index.ts

```diff
@@ -0,0 +1,43 @@
+import * as http from "http";
+import * as url from "url";
+import { StringDecoder } from "string_decoder";
```

Comment:
Here I spent five minutes trying to figure out what string_decoder is an where it comes from, why it is not in the package.json if it is a dependency? Only to find out that it part of node.js. I never used it/never saw it so I didn't know. If you had imported it from 'node:string_decoder' (same as http or url modules) it would have been obvious at first glance - adding 'node:module-name' in the imports helps the reader understand where the dependency comes from.